changelogs/fragments/sub_plugins_dir_change.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - Move sub plugins cli_parsers, fact_diff and validate to `plugins/sub_plugins folder
```

Comment:
This collection still does not seem to use the ansible-base 2.10 sanity tests. Otherwise, this invalid changelog fragment would have made them fail.
```
ERROR: Found 1 changelog issue(s) which need to be resolved:
ERROR: changelogs/fragments/sub_plugins_dir_change.yaml:0:0: (WARNING/2) Inline interpreted text or phrase reference start-string without end-string.
```